### PR TITLE
Issue 59: Http Ingestion How-To

### PR DESCRIPTION
**Pull request by brohitbrose**

### CODE REVIEW

File: src/_guides/http-ingestion.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+There are only three high-level components to this application:
+
+- A `java.net.http.HttpClient` instance, which is shared in parallel among...
+- ...multiple (but bounded) `AgencyAgent` instances who periodically fetch an agency's worth of vehicle data and relay it to...
+- ...multiple (potentially unbounded) number of `VehicleAgents` whose callback functions define the business logic.
+
+## Prerequisites
+
+- Swim server libraries
+- **Java 11+**. This guide takes advantage of `java.net.http.HttpClient`, which is only available starting Java 11.
```
Comment (brohitbrose) [inline src/_guides/http-ingestion.md:23]: Leave a small note saying that others can follow along with HttpUrlConnection? If we declare upfront that Java 11+ is a hard requirement for Swim (which we might already be, I'm not sure), then I don't think we need the note.

**Review by ajay-gov — CHANGES_REQUESTED** [review]
- [x]  Change `Main` class name that wraps the Http Client
- [ ]  Change title of  `NextBus API Wrapper` to `Http API` wrapper, so that the high level topics are generic. Is this ok?
- [ ]  Add a line to indicate that we need to pool since REST APIs need to polled and hence we use timers and how this is a general pattern for request-response based sources like REST
- [ ]  One quesion on the opening system: 
"This guide illustrates how to program a Swim server to ingest data from HTTP sources and subsequently perform business logic."

Should this be: 
This guide illustrates how to develop a Swim application that ingests data from HTTP/REST APIs and instantiate Web Agents that perform business logic?  Same for the Kafka ingestion PR as well. What do you think?

File: src/_guides/http-ingestion.md (hunk 1 above)
Comment (ajay-gov) [reply src/_guides/http-ingestion.md:23]: Ok with enforcing Java 11+ and leaving a note. The note can simply say  that they can use java `HttpClient` library

**Review by ajay-gov — APPROVED** [review]
Looks good!

**Pull request merged**